kodilla-intro/src/main/java/com/kodilla/task25/RandomNumbers.java

```diff
@@ -0,0 +1,47 @@
+package com.kodilla.task25;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RandomNumbers {
+
+    private List<Integer> drawnNumbers = new ArrayList();
+    public static void main(String[] args) {
+        RandomNumbers randomNumbers = new RandomNumbers();
+        randomNumbers.getCountOfRandomNumber();
+
+        System.out.println(randomNumbers.getMaxNumber());
+        System.out.println(randomNumbers.getMinNumber());
+    }
+    public int getCountOfRandomNumber() {
+        Random random = new Random();
+        int result = 0;
+        int sum = 0;
+        while (sum < 5000) {
+            int temp = random.nextInt(31);
+            drawnNumbers.add(temp);
+            sum = sum + temp;
+            result++;
+        }
+        return result;
+    }
+    public int getMaxNumber() {
```

Comment:
te dwie metody nie będą przeglądać całej listy (drogo to kosztuje). Wprowadźmy dwa pol `private int min` oraz `private int max` i ustawmy je inicjalnie na `min = 31` oraz `max = -1`. W każdym kroku pętli w metoddzie z linii 17 sprawdzamy: `if (temp > max) { max = temp; }` oraz `if (temp < min) { min = temp; }`